docs/deployment-manager-guide.md

```diff
@@ -24,3 +24,28 @@ uncheck the **Metadata selection** box and then click **Continue**.
 **WARNING:** Unchecking the **Metadata selection** box is crucial, so that
 `mpdev` does not override the solution metadata configured in earlier steps of
 the Partner Portal wizard.
+
+## Upload solution to Producer Portal (Public Preview)
+
+The script below creates and configures a GCS bucket that you can use to upload
+your Deployment Manager template to Producer Portal.
+
+```
+BUCKET_NAME=<BUCKET_NAME>
+gsutil mb $BUCKET_NAME
+gsutil versioning set on gs://$BUCKET_NAME
+gsutil iam ch "group:marketplace-ops@cloud.google.com:objectViewer" $BUCKET_NAME
+```
+
+Prior to running `mpdev apply`, ensure the `zipFilePath` specified
+in your [`DeploymentManagerTemplate`](../examples/deployment-manager/autogen/singlevm/configurations.yaml)
```

Comment:
Add the `mpdev pkg set` command--reading this comment sorta implies editing the yaml file directly.

Reply:
Done